### PR TITLE
Add Automatic-Module-Name

### LINKED ISSUE #421: Add support for Java Module System (Java9)
From Java 9 release, Java support the Java Module System(Jigsaw).
Legacy library could add 'Automatic-Module-Name' in the MANIFEST.MF
Some instruction: http://branchandbound.net/blog/java/2017/12/automatic-module-name/

### PR DESCRIPTION
**Pull request by A248**
Fixes #421 

I was pleasantly surprised that the maven jar plugin merges `manifestEntries` with the remaining `manifestFile`. The OSGI info is kept too. Automatic-Module-Name is perfectly compatible with the existing build.

### CODE REVIEW

**Review by scholzi100 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -127,11 +127,14 @@
...
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>2.3.2</version>
+        <version>3.2.0</version>
         <configuration>
           <!-- Pick the MANIFEST generated by the bundle plugin -->
           <archive>
             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            <manifestEntries>
+              <Automatic-Module-Name>org.sqlite.jdbc</Automatic-Module-Name>
```
Comment (michael-o) [inline pom.xml:136]: This is wrong because this driver is not developed by SQLite devs.

Comment (A248) [reply pom.xml:136]: You're right, I see. What would you suggest instead? I'm thinking of something like `com.github.xerial.sqlitejdbc`

**Comment by A248** [comment]
Looking for a better module name, so will be converting this to a draft for now.

File: pom.xml (hunk 1 above)
Comment (michael-o) [reply pom.xml:136]: `org.xerial.sqlitejdbc`? Group id plus artifact id.

**Comment by A248** [comment]
Proceeding with org.xerial.sqlitejdbc as the module name

**Comment by onkobu** [comment]
When will this be released? I'd appreciate making jlink happy to have an explicit module. SQLite even more integrated.

**Comment by xerial** [comment]
org.xerial.sqlitejdbc should be ok. I'll merge this change and release a new version soon

**Pull request merged**